### ISSUE TITLE
HTTP Requests logs for rules

### ISSUE THREAD

**Issue by vicgb** [StackStorm/st2#5617]
Hello everyone.

There is some option to follow on streaming events of the type CREATE, DELETE, PUT, etc for a rule, by pointing to.. /stream/v1/stream? I mean, some options for listening into HTTP Requests that affect rules such as a rule has been created or updated or deleted.

What I would like is to see something similar to what you can see with /stream/v1/stream/events=st2.executions... but with any events relative to rules.

Thanks in advance.

**Comment by nzlosh** [comment]
Not that I'm aware of, you'll need to poll the api https://api.stackstorm.com/api/v1/ruleenforcements/

**Comment by vicgb** [comment]
Agreed. I was looking for a way to push from the stackstorm API any HTTP Request instead of doing periodic pulls, so as not to overload the system.
In any case, I'll take a look at it.

Thank you very much :)
